frontend/resourceadm/components/NewPartyListModal/NewPartyListModal.test.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useRef } from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render as rtlRender, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { textMock } from '../../../testing/mocks/i18nMock';
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+import { NewPartyListModal } from './NewPartyListModal';
+import { ServicesContextProps, ServicesContextProvider } from 'app-shared/contexts/ServicesContext';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+
+const mockButtonText: string = 'Mock Button';
+const closeModalMock = jest.fn();
+
+const defaultProps = {
+  org: 'orgname',
+  env: 'tt02',
+  navigateUrl: '/listadmin/partylist/',
+  onClose: closeModalMock,
+};
+
+const mockedNavigate = jest.fn();
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockedNavigate,
+}));
+
+const user = userEvent.setup();
```

Comment:
This should be defined inside each of the ```it``` blocks as per [userEvent´s documentation](https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent) 😄